docs/use-service.md

```diff
@@ -17,68 +17,85 @@ Once you have your Mapzen API key you'll need include it with Terrain Tile reque
 Request a single tile with this URL pattern to get started:
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
```

Comment:
Good point about the S3 file paths. Tracking via https://github.com/tilezen/joerd/pull/151#discussion_r130721596.